src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -212,7 +212,6 @@ public void testValueOf() {
         assertEquals(genericAccount.getEmail(), observedAccountAttributes.getEmail());
         assertEquals(genericAccount.getInstitute(), observedAccountAttributes.getInstitute());
         assertEquals(genericAccount.getCreatedAt(), observedAccountAttributes.createdAt);
-        assertEquals(genericAccount.getStudentProfile(), observedAccountAttributes.studentProfile);
```

Comment:
LHS is a `StudentProfile` object, RHS is a `StudentProfileAttributes` object. They can never be equal. The only reason this passes is because both are `null`.